apps/wallet/src/ui/app/components/address-input/index.tsx

```diff
@@ -17,12 +17,14 @@ export interface AddressInputProps {
     disabled?: boolean;
     placeholder?: string;
     name: string;
+    'data-testid'?: string;
```

Comment:
Why does this need to be a prop? Seems like this can just always be address-input